### DISCUSSION TITLE
ChromeOS recovery for 11th Generation Intel i3?

### DISCUSSION THREAD

**Discussion by sajni8** [sebanc/brunch#1747] [Q&A]
Please someone help me which recovery should I use for Intel i3 11th Generation? Currently I am using Windows 11 but would like to move to ChromeOS as I am invested into Google Ecosystem. I have tried Volteer earlier but I only get blank screen after ChromeOS is booted.

**Comment by spekulant2** [comment]
I used ' dedede ' (beetley) well kernel 5.15 in settings brunch .
